### ISSUE TITLE
"Entry" parameter in "Completion" callback only contains the last word that was typed

### ISSUE THREAD

**Issue by genbod** [jcubic/jquery.terminal#207]
Is there any way to get the whole line instead of the last word?  I would like to be able to tab complete longer commands like: "describe Credentia" etc.  Right now it will only return "Credentia" and even though I have "describe Credentials" in the list that is returned, it obviously doesn't match etc.

**Comment by jcubic** [comment]
I've tested and it work with words that have spaces in them, can you show your code?

**Comment by genbod** [comment]
You are right.  If I have "describe CredentialTreasury", and "describe Fingerprint" in the list that is returned from the completion list and I type: "describe" and then do tab completion, it will list both entries.  However, if I type "describe Cred" and then tab completion it won't list "describe CredentialTreasury"  because the entry is set to "Cred" not "describe Cred".  The solution I have found is to add "CredentialTreasury" and "Fingerprint" to the list as well.  One other issue I found is that no matter what, tab completion tries to turn my entry into a regex object so I can enter file paths like: "yolo\scripts" etc.  I have to type "yolo\\scripts" etc.  Is there any way to disable that?  I don't really need to regex for tab complete especially because when it does tab complete when there are regex pieces it keeps the regex part.  For example: "describe C\*" turns into "describe C\*redentialTreasury".
